css/style.css

```diff
@@ -0,0 +1,101 @@
+body {
+  background-color: darkslateblue;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  line-height: 1.5;
+  color: whitesmoke;
+  margin: 0;
+  padding: 0;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background-image: url(https://img5.goodfon.com/wallpaper/nbig/f/e1/tsvet-forma-fon-230.jpg);
+  background-position: center;
+}
+
+#title {
+  font-size: 2.5rem;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+#title,
+#description {
+  text-align: center;
+}
```

Comment:
-If you have to assignment style CSS for the same tag you can use a class or id and write the code in the same part of the selector for the same class or id.
I strongly suggest you this because makes it easier to make changes to specific elements without affecting others, and also it will be easier to maintain the code and readable for others who review the code. 😉